src/main/java/io/jenkins/plugins/remotingopentelemetry/engine/RemotingSpanExporterProvider.java

```diff
@@ -0,0 +1,23 @@
+package io.jenkins.plugins.remotingopentelemetry.engine;
+
+import io.opentelemetry.sdk.trace.export.SpanExporter;
+import io.opentelemetry.exporter.otlp.trace.OtlpGrpcSpanExporter;
+
+import javax.annotation.Nonnull;
+
+/**
+ * Provides {@link SpanExporter}
+ */
+public class RemotingSpanExporterProvider {
+    /**
+     * @param config to create {@link SpanExporter}
+     * @return configured {@link SpanExporter}
+     */
+    @Nonnull
+    public static SpanExporter create(EngineConfiguration config) {
```

Comment:
Note that in Java, we tend to stay away from static method to help mocking / stubbing for testing...

Reply:
I see. I'll fix it using "Dependency Injection Pattern" later, thanks!